libafl/src/stages/sync.rs

```diff
@@ -86,24 +92,53 @@ where
         state: &mut Z::State,
         manager: &mut EM,
     ) -> Result<(), Error> {
+        log::debug!("Syncing from disk: {:?}", self.sync_dir);
         let last = state
```

Comment:
This could be cleaned up/sped up using `metadata_or_insert_with`, but I guess we can do it in another PR...